metrics/metrics.go

```diff
@@ -0,0 +1,78 @@
+package metrics
```

Comment:
Could you please add the license text also here like in main.go ?

Reply:
license text added